README.md

```diff
@@ -33,17 +33,31 @@ Options:
 
 ### Syntax
 The parser looks for comments on your type defs and parse the enum declarations from it.
-The parser will look for `ENUM(` and continue to look for comma separated values until it finds a `)`.  You can put values on the same line, or on multiple lines.
+The parser will look for `ENUM(` and continue to look for comma separated values until it finds a `)`.  You can put values on the same line, or on multiple lines.\
 If you need to have a specific value jump in the enum, you can now specify that by adding `=numericValue` to the enum declaration.  Keep in mind, this resets the data for all following values.  So if you specify `50` in the middle of an enum, each value after that will be `51, 52, 53...`
 
+
+You can use comments inside enum that start with `//`
```

Comment:
Please add a statement about the `//` needing to be on the same line as the enumerated value, and not the line above it.

Reply:
Done. Thank you.